collections/bitcoin/_posts/2025-07-25-engineering-dialogue.md

```
@@ -0,0 +1,94 @@
+- Writing
+- nostr
+boris_link: https://read.withboris.com/a/naddr1qq2x2mn8d9hx2etjd9hxwttyd9skcmm8w4jszrthwden5te0dehhxtnvdakqz9nhwden5te0wfjkccte9ec8y6tdv9kzumn9wspzpq7enxs5scju854msxd0xpjvpa4p94763rmgktrfyg0n5arpw8geqvzqqqr4gu952xyu
+---
+
+Every time we speak into the void of the internet, we are entering into a dialogue with the world. The world reacts, and even if we can’t see the faces of our audience, their reactions are real. They manifest in the digital realm, in conversations, in shared ideas, in the birth of new projects.
+
+Dialogue is at the core of [Sovereign Engineering](https://sovereignengineering.io/). It is why the program was created, why we made the decisions we made, and why our weekly loop is structured the way it is. It is the heart and the engine of each cohort.
+
+From the get-go, the idea was to create an environment that is conducive to high-bandwidth communication, experimentation, and ideation. The overarching goal was and is one and the same: *implementation*. Creating—and shipping—new freedom tech.
+
+### The Problem of Creation
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)